packages/expo/src/Notifications/Notifications.ts

```diff
@@ -40,7 +40,11 @@ export function emitNotification(notification) {
     }
   }
 
-  _emitter.emit('notification', notification);
+  if (_emitter) {
+    _emitter.emit('notification', notification);
+  } else {
+    throw new Error(`Must have EventEmitter available in DOM`);
+  }
```

Comment:
@sjchmiela  should this just be `_emitter?.emit(...)`?

Reply:
it depend if you want a silent error or keep a trace for later

Reply:
Side question, I am still on expo sdk36, will I be stuck to get this fix?

Reply:
you can use patch-package for that: https://www.npmjs.com/package/patch-package

we don't tend to cherrypick commits to old sdk versions unless they are critical

Reply:
Ok, what do I do for https://github.com/expo/expo/pull/9823#discussion_r472570527 ?

Reply:
Wouldn't it be better to initialize `emitter` if it's not there? How about we add `_maybeInitEmitter` to `emitNotification`? Or create some smart getter for `_emitter` inside the file so we are always sure it exists?

I can see this code being buggy on web, where we register a listener

https://github.com/expo/expo/blob/92f6de58cfc369f1e4c0adee7c616e1e52bcc253/packages/expo/src/Notifications/ExponentNotifications.fx.web.ts#L7-L9

but never ensure that the initializer is there (as it is only initialized once someone adds a notification listener).

Reply:
I don't know since I don't know the background here, should `someone` always have a listener in a soon futur, etc...